### ISSUE TITLE
uMatrix causes error with Google Hangouts regardless of whether uBlock/uMatrix is enabled

### ISSUE THREAD

**Issue by argiopetech** [gorhill/uBO-Extra#15]
Not sure when this became an issue, but it's definitely present in 2.4. On attempting a call through https://hangouts.gooogle.com (just pressing "Video Call" button is sufficient), Chrome notes, "Error - Sorry, we encountered a problem joining this video call". The console shows:

```
(3) WebSocket connection to 'wss://hangouts.google.com/' failed: WebSocket is closed before the connection is established.
```

This issue occurs only when uBO-Extra is enabled, never when it is disabled. The status of uBlock and uMatrix is irrelevant, i.e., uMatrix enabled with uBlock and uMatrix disabled still causes the issue to occur.

**Comment by gorhill** [comment]
It's a warning, normal given what uBO-Extra has to do. Does it prevent Google Hangout from working?

**Comment by gorhill** [comment]
I can't figure what is the issue, as my breakpoints are not triggered. For now I will create exceptions for some of the scriptlets, and I added `hangouts.google.com` to the WebRTC wrapper.

**Comment by IDKwhattoputhere** [comment]
Maybe add `ipleak.net` too so that one can test if WebRTC would leak IP addresses to Google Hangouts?

**Comment by gorhill** [comment]
`ipleak.net` works fine for me, it could detect my local IP address when I disabled the setting in uBO.

**Comment by IDKwhattoputhere** [comment]
That's true, sorry. I must have not disabled it when I tested.

**Comment by argiopetech** [comment]
Thanks for your quick resolution, @gorhill!

**Comment by okiehsch** [comment]
`https://webrtc.github.io/samples` is also broken, for example
`https://webrtc.github.io/samples/src/content/peerconnection/webaudio-input`

1. with `uBO extra 2.3` 

![webrtc](https://cloud.githubusercontent.com/assets/16838044/21584683/6da44aea-d0af-11e6-8993-dc328a9c0038.PNG)

2. with `uBO extra 2.5`

![webrt](https://cloud.githubusercontent.com/assets/16838044/21584748/fa5a8508-d0af-11e6-96eb-e6bc72ce87c3.PNG)

**Comment by gorhill** [comment]
Can you confirmed above is fixed on your side with 2.6?

**Comment by okiehsch** [comment]
Yes `https://webrtc.github.io/samples` is fixed with 2.6.
